lib/dynamoid/persistence.rb

```diff
@@ -507,7 +507,12 @@ def persisted?
     # @return [true|false] Whether saving successful or not
     # @since 0.2.0
     def save(options = {})
-      self.class.create_table(sync: true)
+      if Dynamoid.config.create_table_on_save
+        self.class.create_table(sync: true)
+      else
+        Dynamoid.logger.warn "Table #{self.class.table_name} is not created as create_table_enabled is disabled"
+      end
+
```

Comment:
@andrykonchin What do you think about removing the warn? it's gonna show up on every save. And we have the exception anyway

Reply:
Yeah, agree.

TBH I would consider implicit table creation as an auxiliary feature, that is used in development/test environments and will be disabled on production. So if somebody expects a warning about unusual behaviour - I suppose it should be a warning when a table is created implicitly. But I believe we should rely on user experience and add warning when users ask for it.

Reply:
great, i'll remove the log then